routesrv/polling.go

```diff
@@ -101,12 +86,15 @@ func (p *poller) poll(wg *sync.WaitGroup) {
 			if initialized {
 				logger.Info(LogRoutesInitialized)
 				span.SetTag("routes.initialized", true)
-				routesInitialized.SetToCurrentTime()
+				p.metrics.UpdateGauge("routes_initialized_timestamp", (float64(time.Now().UnixNano()) / 1e9))
```

Comment:
I guess we could have a helper like
```go
func (p *poller) setGaugeToCurrentTime(name string) {
    p.metrics.UpdateGauge(name, (float64(time.Now().UnixNano()) / 1e9))
}
```
to make it a bit cleaner.

Reply:
done